src/main/resources/application.properties

```diff
@@ -228,6 +228,10 @@ quarkus.openshift.env.vars.INDEXING_BULK_SIZE=${INDEXING_BULK_SIZE}
 quarkus.openshift.startup-probe.initial-delay=2M
 quarkus.openshift.startup-probe.period=1M
 quarkus.openshift.startup-probe.failure-threshold=15
+quarkus.openshift.readiness-probe.period=1M
+quarkus.openshift.readiness-probe.failure-threshold=15
+quarkus.openshift.liveness-probe.period=10s
+quarkus.openshift.liveness-probe.failure-threshold=90
```

Comment:
I'm not sure why this should be necessary...

Liveness doesn't even require indexing, so that one doesn't seem to make sense.

Readiness does require indexing, but:

1. On startup, the startup probe prevails, IIUC: https://kubernetes.io/docs/concepts/configuration/liveness-readiness-startup-probes/#startup-probe
2. During the application's life, if readiness suddenly is lost... it would mean the index went down? Because we use rollover, so there is no other reason for the index to become empty suddenly. So I don't see the point of waiting (nor do I see the point of restarting, but well, that's kube for you)

Reply:
removed 👍🏻 

hmm, I noticed this one time: the app wasn't ready and a termination request was sent and then a restart. Currently, the index is empty, and it keeps the pod as "not ready". I'm not sure how it got to the empty index state, my guess is that with the schema change and all the restarts, it messed up somehow 😕.

> On startup, the startup probe prevails, IIUC: https://kubernetes.io/docs/concepts/configuration/liveness-readiness-startup-probes/#startup-probe

nice 😃, I was reading some other doc page and couldn't find this info there 🤦🏻😃